eap-datagrid-subsystem/README.md

```diff
@@ -91,11 +91,32 @@ To see the behaviour for JDG caches
 Notes
 =======
   - You can simple rename the jboss-eap-datagrid-subsystem-AppWeb.war to any other name.war to simulate different applications.
-    If there are multiple applicatations they share the same cache!
+    If there are multiple applications they share the same cache!
   - Caches are independent from application lifecycles. Undeploying any, or all, applications instances has no affect on a cache's lifecycle or data.
     Even if the cache has no persistence the data will be kept as long as the server is not stopped.
   - In this example the EAP server is not clustered, but if the cache is configured as clustered with the datagrid-jgroups subsystem and replicated or 
     distributed caches, the cache will be synchronized between different EAP instances if configured correctly.
   - The respective JGroups subsystem needs to be defined for both EAP and JDG as the respective subsystems depend on different JGroups versions and provide different functionality.
   - Also it is indespensable that the communication addresses and port for the JGroups subsystems need to be different. This prevent from cross-talking
     which cause performance drawback, WARN message or error messages.
+
+
+Run tests
+========================
+1. Prepare the server instances and keep EAP server running
```

Comment:
How do you prepare server instances?

Reply:
@galderz  I have updated the README. Hope now it is clear. Just need to follow steps from upper part of readme :)